cms/templates/js/self-paced-due-date-editor.underscore

```diff
@@ -0,0 +1,16 @@
+<ul class="list-fields list-input datepair date-setter">
+    <li class="field field-text field-due-in">
+        <label for="due_in"><%- gettext('Due in:') %></label>
+        <input type="number" id="due_in" name="due_in" value=""
+            placeholder="" autocomplete="off"/> weeks
```

Comment:
Do you know if you can add a max here as part of the input? Likely still need to validate elsewhere (JS file perhaps?), but could be an easy win to enforce in the HTML

Reply:
After adding the max as part of the input, the user can still manually enter a number greater than the max but it still seems to put some restriction with the options they can put! Will probably create a separate ticket just to deal with this case

Reply:
Yeah, I think that is good and then in the JS that handles the input, you could just do some validation of the number as well and throw an error if they don't respect it. Separate ticket totally sounds good to me though :)